pkg/chunk/storage/utils_test.go

```diff
@@ -33,6 +34,7 @@ func forAllFixtures(t *testing.T, storageClientTest storageClientTest) {
 
 	for _, fixture := range fixtures {
 		t.Run(fixture.Name(), func(t *testing.T) {
+			fmt.Println(fixture.Name())
```

Comment:
Do you need this? The test run with `t.Run(fixture.Name()` and you should already have the fixture name in the tests output.

Reply:
This was for debugging. I'll remove it.